scripts/openstack-quickstart-demosetup

```diff
@@ -1055,6 +1055,16 @@ EOF
         ansible-playbook -i monasca-hosts monasca.yml
     )
 
+    # do the agent setup
+    monasca-setup \
+        -u "admin" \
+        -p "$pw" \
+        --skip_enable \
```

Comment:
Hm. Not Sure. I used the commit from @sjamgade . But we enable the service manually later. So it should be fine.